### ISSUE TITLE
Vagrant Push (via FTP) Not Pushing Anything

### ISSUE THREAD

**Issue by otjutt** [mitchellh/vagrant#7394]
#### Vagrantfile

```
Vagrant.configure(2) do |config|
  config.vm.box = "ubuntu/trusty64"
  config.vm.synced_folder "src/", "/home/vagrant"
  config.push.define "ftp" do |push|
    push.host = "ftp_host_name"
    push.username = "ftp_username"
    push.password = "ftp_password"
    push.passive = true
    push.secure = false
    push.destination = "/"
    push.dir = "C:\projects\project_name\src\dist"
  end
end
```

#### Output

```
PS C:\projects\project_name> vagrant push
Uploading C:/projects/project_name to ftp_host_name@ftp_username:/
PS C:\projects\project_name>
```

#### Expected Behaviour

The files located at `C:\projects\project_name\src\dist\` in my local machine should be uploaded to `/` in the remote server.

#### Actual Behaviour

Dist directory has an `index.html` file. The file is not uploaded.

#### Vagrant Version
```
PS C:\projects\project_name> vagrant --version
Vagrant 1.8.1
PS C:\projects\project_name>
```

#### Other Information

I'm using Windows 10.
If you want me to send `vagrant push --debug` then I can send it, if it is of any use.

**Comment by sethvargo** [comment]
Hi @otjutt 

Could you please run Vagrant in [debug mode](https://www.vagrantup.com/docs/other/debugging.html) and put the output in a GitHub gist and paste the link here? Thanks!

**Comment by otjutt** [comment]
Hi Seth,

Thank you very much for your quick response. I truly appreciate your dedication and commitment. I was away from PC and after returning I saw that file `index.html` is present in my remote server. 

Somehow, while recklessly trying to fix it, I had it fixed in a certain attempt. I literally tried everything again to find the issue. The issue was in using backward slashes instead of forward slashes in `push.dir`.

I changed the following line:-
```
push.dir = "C:\projects\project_name\src\dist"
```
to:-
```
push.dir = "C:/projects/project_name/src/dist/"
OR
push.dir = "./src/dist/"
OR
push.dir = "src/dist/"
OR
push.dir = "src/dist"
```
It works fine now.

Thanks,
Omar
